Ports/.port_include.sh

```diff
@@ -736,6 +736,16 @@ do_generate_patch_readme() {
     >&2 echo "Successfully generated entries for $count patch(es) in patches/ReadMe.md."
 }
 
+do_generate_available_ports() {
+    pushd "${PORT_META_DIR}/.."
+
+    "${PORT_META_DIR}/../generate-available-ports.py" > AvailablePorts.md
```

Comment:
This can be just `./generate-available-ports.py > AvailablePorts.md` I think?

Reply:
definitely!